source/includes/fact-agg-pipeline-merge-out-confirm.rst

```diff
@@ -0,0 +1,7 @@
+Aggregation pipelines can contain operators such as 
```

Comment:
[suggestion] This is a little hard to parse. Consider:

Some aggregation operators, like $merge and $out, can modify your collection's data. 

If your aggregation pipeline contains operators that can modify your collection's data, you are prompted for confirmation before the pipeline is executed.

Reply:
Updated